### ISSUE TITLE
Extra event's popups

### ISSUE THREAD

**Issue by Smolcio** [Mytherin/Tibialyzer#148]
Is there any way to add **my own**  "event popup"? For Example:
I do tasks on hive and I want to show notify while complete some task.
In this time does apear Server message:

1.  _"That was the last antenna to be blinded."_ / _"You have blinded enough of them."_

2. _" That was the last pore that you had to disable."_ / _"You have done enough."_

I think That not everyone would like to use this think but would be very helpfull

**Comment by Mytherin** [comment]
There is currently no easy way of adding events using the client. You can add them to the database by opening `database.db` with a SQLite client program and firing some SQL queries at the database.

First, you need to get the id of a creature (used for the image shown).

```sql
SELECT id,name FROM Creatures WHERE LOWER(name) LIKE "%insect%";
```
You will get something like this. Take the id of the creature you want to have as image for your event (e.g. Insectoid Scout).
```
498|insect swarm
603|insectoid scout
654|Insectoid Worker
```
You can then add a new event to the database using the creature id, the event message and a location.
```sql
INSERT INTO Events (title,location,creatureid) VALUES ('That was the last antenna to be blinded.', 'Gray Beach', 603);
```

Commit your event to the database and reload Tibialyzer. It should then recognize the new event.

**Comment by Smolcio** [comment]
Thakns for quick answer. I'll try to find way to edit database. I just have to get editor. Any idea witch could be best?

**Comment by Mytherin** [comment]
You can use [SQLite Browser](https://github.com/sqlitebrowser/sqlitebrowser) if you want a GUI tool; if you want a command line tool you can download the tools from the official SQLite website.

**Comment by Smolcio** [comment]
I made it in that way:

1.  I opened Data Structure

2. Underline _Event Messages_ and _Browse Table_
 - set Event ID (example 64) 
 - set Message (witch appear in Server Log) _That did it! You have hit the last necessary digestion bubble._

3. Underline _Events_ and  _Browse Table_
 -set same Event ID (in this way 64) 
 -Title similar as message 
 -location set as _Gray Beach_
 -creature set on 658 (Floor Bloob)

After that I wrote changes and turned on Tibialyzer. Everything is work properly

Thanks for helping and I hope that this posts will helt to someone
